common/src/WooCommerce.php

```diff
@@ -204,4 +210,14 @@ private function call_method($obj, $name) {
         }
         return @$method->invoke($obj);
     }
+
+    private function statusReached(string $new_status): bool {
+        $selected_statuses = get_option($this->plugin->getOptionName('order_statuses')) ?: ['wc-completed'];
```

Comment:
You should use `get_default_config()['order_statuses']` here, or make that a class constant (better), so that you don't repeat `['wc-completed']`.